ruff.toml

```diff
@@ -100,4 +100,4 @@ exclude = ["tests/**/invalid_file*"]
 # Some of our helpers have deliberatly the name of a standard library module
 
 [lint.flake8-builtins]
-builtins-allowed-modules = ["math", "pickle", "string"]
+builtins-allowed-modules = ["math", "pickle", "string", "html"]
```

Comment:
Also, I think you don't need this change anymore because I fixed something similar in a different way in a different branch. Try reverting it and see if it works.